spring-cloud-starter-task-composedtaskrunner/README.adoc

```diff
@@ -0,0 +1,134 @@
+//tag::ref-doc[]
+= Composed Task Runner
+
+A task that executes a tasks in a directed graph as specified by a DSL that is
+passed in via the `--graph` command line argument.
+
+== Overview
+The Composed Task Runner parses the graph DSL and for each node in the graph it
+will execute a restful call against a specified http://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/[Spring Cloud Data Flow]
+instance to launch the associated task definition.  For each task definition that is executed the
+Composed Task Runner will poll the database to verify that the task completed.
```

Comment:
Created issue for this: https://github.com/spring-cloud-task-app-starters/composed-task-runner/issues/4